views/results.ejs

```diff
@@ -2,4 +2,14 @@
 
 <h1>These people do:</h1>
 
+<form>
```

Comment:
Forms are used to submit things. You're not submitting any data so no need for a form element.